src/form.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable import/extensions */
```

Comment:
_[OPTIONAL]_

Not a best practice to disable linters rules as these rules are helping to write better code